src/System.ComponentModel.Composition/System.ComponentModel.Composition.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
```

Comment:
Could you please normalize the solution/projects by running https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md#updating-configurations?

Reply:
sln normalized